ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -303,15 +310,20 @@ export const NetworkListMenu = ({ onClose }) => {
         onClick={() => {
           dispatch(toggleNetworkMenu());
           dispatch(setActiveNetwork(network.providerType || network.id));
-
-          // If presently on and connected to a dapp, communicate a change to
-          // the dapp via silent switchEthereumChain that the network has
-          // changed due to user action
+          if (process.env.CHAIN_PERMISSIONS) {
+            grantPermittedChain(selectedTabOrigin, network.chainId);
+            if (!flattenedPermittedChains.includes(network.chainId)) {
+              dispatch(showPermittedNetworkToast());
```

Comment:
I don't believe I'm seeing this toast on latest?

Reply:
https://github.com/user-attachments/assets/ca492a27-65a0-4809-b279-84679248f816

Reply:
This toast is supposed to show up on the home page when the user grants permission to an enabled network. It seems to be working for me

https://github.com/user-attachments/assets/26243db8-94dc-490e-9870-6a5f6e907c24

Reply:
hmmm yeah I expected to see it there but did not